web/client/codechecker_client/blame_info.py

```diff
@@ -1,9 +1,9 @@
 import json
 import os
-import sys
 import zipfile
 
-from concurrent.futures import ProcessPoolExecutor
+import multiprocess
+
 from git import Repo
 from git.exc import InvalidGitRepositoryError
 from typing import Dict, Iterable, Optional
```

Comment:
```suggestion
import os
from typing import Dict, Iterable, Optional
import zipfile

from git import Repo
from git.exc import InvalidGitRepositoryError
import multiprocess
```

`typing` is part of the Python STL, while `multiprocess` is a 3PP library just like PyGit.